docs/guides/how_to_guides/creating_batches/how_to_load_a_pandas_dataframe_as_a_batch.rst

```diff
@@ -1,18 +1,145 @@
+
+            .. code-block:: yaml
+
+                insert_your_pandas_datasource_name_here:
+                  class_name: Datasource
+                  execution_engine:
+                    class_name: PandasExecutionEngine
+                  data_connectors:
+                    insert_your_runtime_data_connector_name_here:
+                      module_name: great_expectations.datasource.data_connector
+                      class_name: RuntimeDataConnector
+                      runtime_keys:
+                        - some_key_maybe_pipeline_stage
+                        - some_other_key_maybe_run_id
+
+
+        2. Obtain an Expectation Suite
+
+            .. code-block:: python
+
+                suite = context.get_expectation_suite(insert_your_expectation_suite_name_here)
+
+            Alternatively, if you have not already created a suite, you can do so now.
+
+            .. code-block:: python
+
+                suite = context.create_expectation_suite(insert_your_expectation_suite_name_here)
+
+        3. Construct a BatchRequest
+
+            We will create a BatchRequest and pass it our DataFrame via the ``batch_data`` argument.
+
+            Attributes inside the ``partition_request`` are optional - you can use them to attach additional metadata to your DataFrame. When configuring the Data Connector, you used ``runtime_keys`` to define which keys are allowed.
+
+            NOTE: data_asset_name can be set only to this predefined string: ``“IN_MEMORY_DATA_ASSET”`` for now. We will fix it very soon and will allow you to specify your own name.
+
+            .. code-block:: python
+
+                from great_expectations.core.batch import BatchRequest
+
+                batch_request = BatchRequest(
+                    datasource_name="insert_your_pandas_datasource_name_here",
+                    data_connector_name="insert_your_runtime_data_connector_name_here",
+                    batch_data=insert_your_dataframe_here,
+                    data_asset_name="IN_MEMORY_DATA_ASSET",
+                    partition_request={
+                        "partition_identifiers": {
+                            "some_key_maybe_pipeline_stage": "ingestion step 1",
+                            "some_other_key_maybe_run_id": "run 18"
+                        }
+                    }
+                )
+
+
+        4. Construct a Validator
+
+            .. code-block:: python
+
+                my_validator = context.get_validator(
+                    batch_request=batch_request,
+                    expectation_suite=suite
+                )
+
+        Now that you have a Validator, you can use it to create Expectations or validate the data.
```

Comment:
We can run my_validator.active_batch.head() here to demo that the batch contains the data.

Reply:
+1